pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -504,15 +507,21 @@ func (l LibvirtWrapper) SetupLibvirt() (err error) {
 		return err
 	}
 
-	if envVarValue, ok := os.LookupEnv("LIBVIRT_DEBUG_LOGS"); ok && (envVarValue == "1") {
+	var libvirtLogVerbosityEnvVar *string
+	if envVarValue, envVarDefined := os.LookupEnv(services.ENV_VAR_VIRT_LAUNCHER_LOG_VERBOSITY); envVarDefined {
+		libvirtLogVerbosityEnvVar = &envVarValue
+	}
+	_, libvirtDebugLogsEnvVarDefined := os.LookupEnv(services.ENV_VAR_LIBVIRT_DEBUG_LOGS)
```

Comment:
I'd probably move this hunk into `getLibvirtLogFilters()` too, since the environment variables are not used in `SetupLibvirt()` except for passing them to that function.

Other than that, everything looks great as far as I can tell! :)

Reply:
To be clear, the current version is fine too! I just think moving the hunk would make the code a bit cleaner, but it's not really something that I would consider mandatory :)

Reply:
The thing is that it would be impossible to unit test it this way (or we would need to define actual environment variable which is.. dirty).

Another way to look at it is that now `getLibvirtLogFilters()` is very cohesive - it gets all the data, the only one making the decision and returning the right filters and whether to use them. `SetupLibvirt()`, on the other hand, writes files, fetches environment variables, and basically interacts with the outer world.

Reply:
Right, I didn't consider that angle (and didn't notice you had added unit tests for the function). Please disregard my comment then!

Reply:
:smiley: 
Thanks for your great comments :+1: